documentation/tutorials/get-started.md

```diff
@@ -71,24 +72,42 @@ Open the project in your text editor, and we'll get started.
 
 Add the ash dependency to your `mix.exs`
 
-{{mix_dep:ash}}
+```elixir
+defp deps do
+  [
+    # {:dep_from_hexpm, "~> 0.3.0"},
+    # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
+    {{mix_dep:ash}}, # <-- add this line
```

Comment:
This may not work with ash_hq because it's in a code block? Let's merge it and I'll see.

Reply:
If anything could replace with 
```
# add the line below
{{mix_dep:ash}},
```

or is it that I needs to be a block on it's own?